modules/govuk/manifests/apps/ckan/paster_cronjob.pp

```diff
@@ -26,13 +26,14 @@
   $weekday = undef,
   $plugin = undef,
   $paster_command = undef,
+  $ckan_config = '--config=/var/ckan/ckan.ini',
```

Comment:
This would probably be better described as `extra_args`, or something similar? Maybe it also needs to default to `''`.

Reply:
Or rather, the behavior may be clearer if the default is `''`.

Reply:
The default is `--config=/var/ckan/ckan.ini`, this is a special case of running the paster command that doesn't want the `config` args as it stops the process from running, most paster commands need the default ckan config.

Reply:
Ah, ok, I didn't realise this class was for configuring a ckan specific job (but I see that now).